docs/markdown/snippets/response-files.md

```diff
@@ -0,0 +1,7 @@
+## Response files enabled on Linux, reined in on Windows
+
+Meson used to always use response files on Windows,
+but never on Linux.
+
+It now strikes a happier balance, using them on both platforms,
+but only when needed to avoid commandline length limits.
```

Comment:
`commandline` should be hyphenated or two words.

Reply:
Done.